tracing/opencensus.go

```diff
@@ -114,7 +115,14 @@ func createOCTConfig(cfg *config.Config) *trace.Config {
 // WithExporter returns a ConfigOption for use with NewOpenCensusTracer that configures
 // it to export traces based on the configuration read from config-tracing.
 func WithExporter(name string, logger *zap.SugaredLogger) ConfigOption {
+	return WithExporterFull(name, name, logger)
+}
+
+// WithExporterFull supports hostPort argument, which is used for the endpoint.
+// The hostPort allows HOST:PORT or HOST. If no port is specified, port 80 will be used.
+func WithExporterFull(name, hostPort string, logger *zap.SugaredLogger) ConfigOption {
 	return func(cfg *config.Config) error {
+		const defaultPort = ":80"
```

Comment:
What is the significance of this `80`? Is it ever correct in the context of a pod?

Reply:
The `:80` is for compatibility. Current client code calls `WithExporterFull()` without using port but [fixed :80 is appended](https://github.com/knative/pkg/pull/1353/files#diff-a95c97ba65048dbcfd15b2d2b54b4a0dL143) so I thought we should keep using it.